app/komponenter/confirmationPanel.tsx

```diff
@@ -0,0 +1,38 @@
+import { ConfirmationPanel } from '@navikt/ds-react';
+import { useState } from 'react';
+import { Button } from '@navikt/ds-react';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from './tekstBlokk/tekstBlokk';
+
+interface BekreftelsePanelProp {
+  melding: SanityDokument | undefined;
+  samtykke: SanityDokument | undefined;
+  feilmelding: SanityDokument | undefined;
+}
+
+const BekreftelsePanel: React.FC<BekreftelsePanelProp> = ({
+  melding,
+  samtykke,
+  feilmelding,
+}: BekreftelsePanelProp) => {
+  const [lest, settLest] = useState(false);
```

Comment:
```suggestion
  const [samtykkeErBekreftet, settSamtykkeErBekreftet] = useState(false);
```
Synes dette er mer beskrivende